openapi-cli/src/test/java/org/ballerinalang/openapi/CodeGeneratorTest.java

```diff
@@ -147,6 +147,58 @@ public void generateSkeletonForRequestbody() {
         }
     }
 
+    @Test(description = "Test Ballerina skeleton generation for multiple Path parameter")
+    public void generateSkeletonForTwoPathParameter() {
+        final String serviceName = "openapipetstore";
+        String definitionPath = RES_DIR.resolve("multiPathParam.yaml").toString();;
+        CodeGenerator generator = new CodeGenerator();
+
+        try {
+            String expectedServiceContent = getStringFromGivenBalFile(expectedServiceFile, "generated_bal.bal");
+            generator.generateService(definitionPath, definitionPath, definitionPath, serviceName,
+                    resourcePath.toString(), filter);
+            if (Files.exists(resourcePath.resolve("openapipetstore-service.bal"))) {
+                String generatedService = getStringFromGivenBalFile(resourcePath, "openapipetstore-service.bal");
```

Comment:
bal file names need to use underscore for word separation. `openapipetstore_service.bal`